dwitter/templates/dweet/dweet.html

```diff
@@ -111,11 +111,15 @@
   <canvas id=c></canvas>
   <div class=error-display></div>
   <script>
-    var stats = new Stats();
-    stats.showPanel(0);
-    stats.dom.setAttribute('id', 'stats');
-    document.body.appendChild(stats.dom);
+    function createStats() {
+      var stats = new Stats();
+      stats.showPanel(0);
+      stats.dom.setAttribute('id', 'stats');
+      document.body.appendChild(stats.dom);
+      return stats;
+    }
 
+    var stats = createStats();
```

Comment:
If `new Stats()` in `createStats` fails, `stats` will be `undefined`. It'll then be reinitialized once the element that contains it is visible, in `loop`.